tests/testing/make_sdists.py

```diff
@@ -6,12 +6,13 @@
 def random_string():
     """return a short suffix that shouldn't collide with any subsequent calls"""
     import os
-    import base64
+    import string
+    import random
 
-    return '.'.join((
-        str(os.getpid()),
-        base64.urlsafe_b64encode(os.urandom(3)),
-    ))
+    return '{}.{}'.format(
+        os.getpid(),
+        ''.join(random.choice(string.ascii_lowercase) for x in range(8)),
```

Comment:
You're right, I'm confused somehow -- for some reason I thought `urlsafe_b64encode` wanted a string as an argument (and not bytes), but that's wrong. I definitely tested this yesterday so I must have gotten confused somewhere else...